react_main/src/components/Roles.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useContext, useRef } from "react";
 import axios from "axios";
 
-import { UserContext, SiteInfoContext, PopoverContext, GameContext } from "../Contexts";
+import { UserContext, SiteInfoContext, PopoverContext } from "../Contexts";

```

Comment:
removed toggleroleprediction from this file

toggleroleprediction opens the role marker pop up